src/nrnpython/nrnpy_hoc.h

```diff
@@ -16,7 +16,9 @@ static PyType_Slot nrnpy_HocObjectType_slots[] = {
     {Py_tp_doc, (void*) hocobj_docstring},
     {Py_nb_bool, (void*) hocobj_nonzero},
     {Py_sq_length, (void*) hocobj_len},
+    {Py_mp_subscript, (void*) hocobj_slice_getitem},
     {Py_sq_item, (void*) hocobj_getitem},
```

Comment:
I thought we decided `Py_mp_subscript` overrides `Py_sq_item`. Is there any reason to keep the latter around?